pages/login.page.js

```diff
@@ -0,0 +1,38 @@
+import BasePage from "./base.page";
+const PAGE_TIMEOUT = 10000;
+const BASE_PAGE = BasePage;
+
+class LoginPage {
+    
+    get usernameInput () { return $('#username'); }
+    get passwordInput () { return $('#password'); }
+    get rememberMe () { return $('#remember-me'); }
+    get logInButton () { return $('#submit'); }
+    get failedLoginMessage () {return $('[class= "badge badge-pill badge-danger"]'); }
+
+    async navigateToPage(){
+        await BASE_PAGE.open('login');
```

Comment:
After you make this class inherit from BasePage, you gain access to super-class' methods and attributes, and you can call them by prefixing **super** to the method call. In this case, it'll be something like:

```javascript
await super.open('login')
```

**super** holds a reference to the superclass of any class that uses it as part of their method / attributes declaration

See reference: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/super